src/SFA.DAS.Forecasting.Web/Views/Estimation/AddApprenticeships.cshtml

```diff
@@ -74,36 +73,40 @@
                         </div>
 
 
-                        <div class="form-group">
+                        <div class="form-group form-group-error">
```

Comment:
Are these hard coded error classes done for a reason?

Reply:
This was added to implement the error validation in this page.  Someone has checked in the code without implementing the functionality in the backend.  I shall speak to Chris about this.